### PR TITLE
LPS-129790 correct the mvcPath and force redirect on delete

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DeleteDataDefinitionMVCActionCommand.java [hunk 1]
```diff
@@ -73,12 +75,26 @@ protected void doProcessAction(
 				themeDisplay.getUser()
 			).build();
 
-		for (long deleteDataDefinitionId : deleteDataDefinitionIds) {
-			dataDefinitionResource.deleteDataDefinition(deleteDataDefinitionId);
+		try {
```
Comment (AliciaGarciaGarcia) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DeleteDataDefinitionMVCActionCommand.java:78]: @4lejandrito what do you think about this solution?

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

File: modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry_metadata_sets.jsp [hunk 2]
```diff
@@ -28,7 +28,7 @@ DLViewFileEntryMetadataSetsDisplayContext dLViewFileEntryMetadataSetsDisplayCont
 />
 
 <portlet:actionURL copyCurrentRenderParameters="<%= true %>" name="/document_library/delete_data_definition" var="deleteDataDefinitionURL">
-	<portlet:param name="mvcPath" value="/view_file_entry_metadata_sets.jsp" />
+	<portlet:param name="mvcPath" value="/document_library/view_file_entry_metadata_sets.jsp" />
```
Comment (4lejandrito) [inline modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view_file_entry_metadata_sets.jsp:31]: I think you can remove this line.

**Comment by 4lejandrito** [comment]
Hi @AliciaGarciaGarcia,

Should we fix this issue in `journal-web` too? They should experience it too. If they have fixed it maybe you can see how they did it.

**Comment by AliciaGarciaGarcia** [comment]
@4lejandrito I think the issue is with the redirection for the navigation pages, and we have a strange behavior with metadata sets....
They do not have it, or I couldn't reproduce it (they delete the WC if the structure is deleted)

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**